src/thirdparty/llvm_dwarf/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+# LLVM sources have warnings, ignore them.
```

Comment:
Do we want to use LLVM's preferred CXXFLAGS?
On my system, that's
`-std=c++14   -fno-exceptions -fno-rtti -D_GNU_SOURCE -D_DEBUG -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS`

Reply:
Sounds good - is there a simple way to find the corresponding windows flags too?

Reply:
I used `llvm-config`. On my windows machine it's `-std:c++14   /EHs-c- /GR- -D_CRT_SECURE_NO_DEPRECATE -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE -D_CRT_NONSTDC_NO_WARNINGS -D_SCL_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_WARNINGS -DUNICODE -D_UNICODE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS`

Reply:
(and mac is identical to Linux above)